dashboard/views.py

```diff
@@ -30,6 +34,10 @@
 GRADE_LOW="low_grade"
 NO_GRADE_STRING = "NO_GRADE"
 
+# string for file type
+RESOURCE_TYPE_STRING = "resource_type"
+RESOURCE_TYPES = utils.get_resource_types()
```

Comment:
If you made the settings change I mention above I'd just use the settings.RESOURCE_TYPES directly.